app/src/main/java/com/dessalines/thumbkey/keyboards/NumericMessagEase.kt

```diff
@@ -26,46 +26,18 @@ val NUMERIC_KEYBOARD_MESSAGEASE =
                         ),
                     swipes =
                         buildMap {
-                            put(
-                                SwipeDirection.LEFT,
-                                KeyC(
-                                    display = KeyDisplay.TextDisplay("≤"),
-                                    action = KeyAction.CommitText("≤"),
-                                    color = ColorVariant.MUTED,
-                                ),
-                            )
                             put(
                                 SwipeDirection.BOTTOM_LEFT,
                                 KeyC(
                                     display = KeyDisplay.TextDisplay("$"),
                                     action = KeyAction.CommitText("$"),
-                                    color = ColorVariant.MUTED,
-                                ),
-                            )
-                            put(
-                                SwipeDirection.TOP,
-                                KeyC(
-                                    // em dash/long minus
-                                    display = KeyDisplay.TextDisplay("—"),
-                                    action = KeyAction.CommitText("—"),
-                                    color = ColorVariant.MUTED,
-                                ),
-                            )
-                            put(
-                                SwipeDirection.TOP_RIGHT,
-                                KeyC(
-                                    // en dash/long minus
-                                    display = KeyDisplay.TextDisplay("–"),
-                                    action = KeyAction.CommitText("–"),
-                                    color = ColorVariant.MUTED,
                                 ),
```

Comment:
why would you remove this?

Reply:
it's added in `1` swipe right, like what MessagEase has

Reply:
These are different characters NB. EN, EM dash and minus  also same q applise to the less than equal sign

Reply:
@asdkant do you mind putting these symbols back please?